src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const COST_PER_DAY = 40;
+
+  const DISCOUNT_LVL1 = 20;
+  const DISCOUNT_LVL2 = 50;
+
+  const DISCOUNT_LVL1_MIN_DAYS = 3;
+  const DISCOUNT_LVL2_MIN_DAYS = 7;
+
+  const sum = days * COST_PER_DAY;
+
+  switch (true) {
+    case days >= DISCOUNT_LVL2_MIN_DAYS:
+      return sum - DISCOUNT_LVL2;
+    case days >= DISCOUNT_LVL1_MIN_DAYS:
+      return sum - DISCOUNT_LVL1;
+    default:
+      return sum;
+  }
```

Comment:
you have code duplication here and also there is a more concise and proper way to do this logic :)

Reply:
ok, let's have some fun with improved solution :)